pipelines/ApVerifyWithFakes.yaml

```diff
@@ -27,16 +27,41 @@ tasks:
         class: lsst.pipe.tasks.imageDifference.ImageDifferenceTask
         config:
             connections.fakesType: "fakes_"
+            # TODO: redundant connection definitions workaround for DM-30210
+            # TODO: move hardcoding of "deep" to dataset configs in DM-26140
+            connections.coaddName: deep
```

Comment:
Parts of it are. Anything with `fakes_` needs to be updated, and since that forces the config to be `deep`-specific, I figured I should redefine `coaddName` so that it's at least self-consistent.

Reply:
Ah, maybe just be explicit about that here, so that one doesn't have to look at the tickets. `Force "deep" to ensure fakes processing overrides whatever the default is` or something like that.